Agents/MetOfficeAgent/.vscode/launch.json

```diff
@@ -2,48 +2,84 @@
     // Use IntelliSense to learn about possible attributes.
     // Hover to view descriptions of existing attributes.
     // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
-    //                          and https://code.visualstudio.com/docs/python/debugging
     "version": "0.2.0",
-    "configurations": [      
-
+    "configurations": [
         {
-            "name": "Python: Current File",
             "type": "python",
-            "request": "launch",
-            "program": "${file}",
-            "console": "integratedTerminal"
+            "name": "Debug",
+            "request": "attach",
+            "host": "localhost",
+            "port": "${input:debug.port}",
+            "preLaunchTask": "compose-deploy",
+            "pathMappings": [
+                {
+                    "localRoot": "${workspaceFolder}/metoffice",
+                    "remoteRoot": "/app/metoffice"
```

Comment:
is it possible to not use the path that is specific to your project so that people can just copy this file into their project?

Reply:
I meant "/app/metoffice", surely this should not be copied over to other projects

Reply:
> I meant "/app/metoffice", surely this should not be copied over to other projects

It reflects the Docker image path; hence, when refactoring those ones, this should of course be adjusted. However, might be tricky to make this very generic